packages/components/src/whats-new/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Guide, GuidePage } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const WhatsNewGuide = () => {
+	const [ isLoaded, setIsLoaded ] = useState( false );
```

Comment:
Do we need the `isLoaded` at all? Can't we just use `whatsNewData` and set it to `null` by default or something? Then if it's an array, continue with rendering below?